python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -993,10 +1001,24 @@ def getFQID(self, sep=None):
             fqid = [self.id]
         else:
             return None
-        cur = stripProxy(self)._getParent()  # FIXME: or use master attribute?
-        while cur:
+
+        try:
+            if stripProxy(self).master is not None:
+                cur = stripProxy(self).master  # FIXME: or use master attribute?
+            else:
+                cur = None
+        except AssertionError:
```

Comment:
Is this really raising an `AssertionError`? Do you know the origin of the assertion?